### DISCUSSION TITLE
Deleting assignment from Classroom does not delete repository copy of original starter code...

### DISCUSSION THREAD

**Discussion by smurray-mvla** [community/community#134180] [GitHub Education]
### Hi everyone,

When I delete an assignment in Classroom, it deletes all associated forked student repositories, but does not delete the
actual repository that they were forked from.

**Comment by boyland** [comment]
I think that's because the repository that was forked was created prior to github classroom -- github classroom didn't create it.

**Comment by smurray-mvla** [comment]
John:

    I am not sure that I agree with that analysis. The source repository
predates the changes to github classroom -
but when I created the assignment - the forked assignment was created *by
github classroom. *

    I would expect that when I delete the assignment from Github classroom
for that particular class, it would also delete
the forked copy that github classroom created.

     Later this week, I will reconfirm that this is indeed what I am seeing
- but I cannot understand why the age/source of the
template repository should matter in the current github classroom
assignment flow.

Regards,

Scott Murray

**Comment by boyland** [comment]
Oh!  Good point.  I only became aware of the latest twist in the github classroom saga after commenting on your post.  I agree with you that the new repository created when the assignment was created should be removed when the assignment was removed.

And, incidentally all the turmoil with github classroom has now convinced me to stop using it.  Next I will be writing a script to create repositories for all my students using the github REST API.
